datafusion/core/tests/sql/explain_analyze.rs

```diff
@@ -696,7 +696,7 @@ async fn test_physical_plan_display_indent() {
         "                CoalesceBatchesExec: target_batch_size=4096",
         "                  FilterExec: c12@1 < 10",
         "                    RepartitionExec: partitioning=RoundRobinBatch(9000)",
-        "                      CsvExec: files=[ARROW_TEST_DATA/csv/aggregate_test_100.csv], has_header=true, limit=None, projection=[c1, c12]",
+        "                      CsvExec: files=1:[[ARROW_TEST_DATA/csv/aggregate_test_100.csv]], has_header=true, limit=None, projection=[c1, c12]",
```

Comment:
I tried several variations on this formatting to show the file names as well as the number of partitions

Other thoughts?
maybe adding a brace and a key?


```suggestion
        "                      CsvExec: files={groups:1, [[ARROW_TEST_DATA/csv/aggregate_test_100.csv]]}, has_header=true, limit=None, projection=[c1, c12]",
```

Reply:
Since you only spell out `groups` once, I think it could be very helpful to spell this out.